File: typescript/src/models/fulfillmentOptionDigital.ts
// this file is @generated

export interface FulfillmentOptionDigital {
  id: string;
  title: string;
  subtitle?: string | null;
  subtotal: number;
  tax: number;
  total: number;
}

export const FulfillmentOptionDigitalSerializer = {
  _fromJsonObject(object: any): FulfillmentOptionDigital {
    return {
      id: object["id"],
      title: object["title"],
      subtitle: object["subtitle"],
      subtotal: object["subtotal"],
      tax: object["tax"],
      total: object["total"],
    };
  },

  _toJsonObject(self: FulfillmentOptionDigital): any {
    return {
      id: self.id,
      title: self.title,
      subtitle: self.subtitle,
      subtotal: self.subtotal,
      tax: self.tax,
      total: self.total,
    };
  },
};

File: rust/src/models/fulfillment_option_digital.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
pub struct FulfillmentOptionDigital {
    pub id: String,
    pub title: String,
    #[serde(skip_serializing_if = "Option::is_none")]
    pub subtitle: Option<String>,
    pub subtotal: i32,
    pub tax: i32,
    pub total: i32,
}
